build/commands/lib/config.js

```diff
@@ -292,6 +293,16 @@ Config.prototype.buildArgs = function () {
     ...this.extraGnArgs,
   }
 
+  if (process.platform === 'darwin' && this.targetOS != 'ios' && args.is_official_build && !this.shouldSign()) {
```

Comment:
not sure how `!this.shouldSign()` ended up in the uplift because it was removed from the original PR because it's not correct and is causing problems https://bravesoftware.slack.com/archives/C7VLGSR55/p1643325085370429?thread_ts=1643323803.539619&cid=C7VLGSR55

Reply:
Not sure how this got here either - but just wanted to share that after discussion, this was addressed in https://github.com/brave/brave-core/pull/12040